app/controllers/pieces_controller.rb

```diff
@@ -7,7 +7,7 @@ def show
   def update
     @selected_piece = Piece.find(params[:id])
```

Comment:
If having trouble with errors here. I recommend putting a `binding.pry`. Then checking to see what is being passed to the controller.

Can run `params` to see what you're getting. Make sure all the data you expect is getting passed